recipe/meta.yaml

```diff
@@ -51,29 +42,29 @@ outputs:
         - nlohmann_json
         - cpp-expected
         - reproc-cpp 14.2.4
-        - spdlog
+        - reproc 14.2.4
+        - spdlog 1.11.0
         - yaml-cpp
         - termcolor-cpp
         - cli11
-        - fmt
-        - libiconv                               # [osx]
-        - winreg                                 # [win]
+        - fmt 9.1.0
+        - winreg             # [win]
       run:
         - libsolv >=0.7.22
```

Comment:
If we want to be explicit, the run section should follow the run_exports.
E.g. - libsolv >=0.7.22<0.8